### PR TITLE
Update collections doc

### PR DESCRIPTION
**Pull request by scottyoung**
_Issue #, if available:_

_Description of changes:_
Users have reported the Collections page (https://docs.amplify.aws/console/uibuilder/collections/) is out of date and shows options and workflows that do not exist in the product.  This commit updates the screenshots and replaces dynamic/specific records with filter and sort conditions.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/pages/console/uibuilder/collections.mdx [hunk 1]
```diff
@@ -11,22 +11,23 @@ Amplify Studio offers a visual way to bind collections to items in your data mod
 
 1. Create a collection from a component
 2. Link collection to a data model
-3. Define a data set with either a dynamic query or specific records.
-4. Components that already have data model bindings, automatically map data to child elements. For components that do not have data model bindings, 
+3. Set filters to customize what data should be displayed
+4. Set sort conditions to customize how the data should be displayed
+5. Components that already have data model bindings, automatically map data to child elements. For components that do not have data model bindings,
```
Comment (renebrandel) [inline src/pages/console/uibuilder/collections.mdx:16]:
```suggestion
> **Note:** If your component doesn't have a property backed by a data model, then you need to manually map a component's property to a designated data model field.
```

File: src/pages/console/uibuilder/collections.mdx [hunk 2]
```diff
@@ -11,22 +11,23 @@ Amplify Studio offers a visual way to bind collections to items in your data mod
...
+4. Set sort conditions to customize how the data should be displayed
+5. Components that already have data model bindings, automatically map data to child elements. For components that do not have data model bindings, 
 
 ![Screenshot showing data binding](/images/console/ui-collections-databind.png)
 
-### Dynamic query
+### Set filters
 
-A dynamic query allows you to sort and filter data from the content view to show relevant data.
+Filters help control what records can be displayed in a collection
```
Comment (renebrandel) [inline src/pages/console/uibuilder/collections.mdx:22]:
```suggestion
Filters control what records can be displayed in a collection:
```

File: src/pages/console/uibuilder/collections.mdx [hunk 3]
```diff
@@ -11,22 +11,23 @@ Amplify Studio offers a visual way to bind collections to items in your data mod
...
 
-![Screenshot showing dynamic query binding](/images/console/ui-collections-dynamicquery.png)
+![Screenshot showing setting filters](/images/console/ui-collections-set-filters.png)
 
-### Specific records
+### Set sort conditions
 
-Link the collection to specific items in your database.
+Sort conditions help control the direction records can be displayed in a collection
```
Comment (renebrandel) [inline src/pages/console/uibuilder/collections.mdx:28]:
```suggestion
Sort conditions control the direction records are displayed in a collection.
```

**Review by renebrandel — APPROVED** [review]

**Review by rachelnabors — APPROVED** [review]

**Review by timngyn — APPROVED** [review]

**Comment by rachelnabors** [comment]
2020_me_thanks

**Pull request merged**